packages/strapi-parts/src/MainNav/MainNav.js

```diff
@@ -7,8 +7,10 @@ import { MainNavContext } from './MainNavContext';
 const MainNavWrapper = styled(Grid)`
   width: ${({ condensed }) => (condensed ? 'max-content' : `${224 / 16}rem`)};
   background: ${({ theme }) => theme.colors.neutral0};
-  height: 100%;
-  position: relative;
+  position: sticky;
+  top: 0;
+  height: 100vh;
+  z-index: 2;
```

Comment:
ℹ️ This is necessary, else, the nav condense button goes under the sub nav bar that has a position sticky and that is positionned after in the DOM (meaning the subnavbar would be above)